pkg/provider/resource_cluster.go

```diff
@@ -383,6 +384,11 @@ func (c *clusterResource) Schema(ctx context.Context, req resource.SchemaRequest
 				ElementType:         types.StringType,
 				PlanModifiers:       []planmodifier.Set{setplanmodifier.UseStateForUnknown()},
 			},
+
+			"superuser_access": schema.BoolAttribute{
+				MarkdownDescription: "Is superuser access enabled.",
+				Optional:            true,
```

Comment:
This should be a computed field only if we don't set it in the request and expect it in the response

Reply:
this is a boolean value which user can set to enable/disable

Reply:
maybe we can update the description to me more specific

Reply:
I see. In that case we should have both optional: true and computed: true. So we can set it and also expect a value back

Reply:
done